webrender/res/brush_mix_blend.glsl

```diff
@@ -203,17 +204,17 @@ const int MixBlendMode_Saturation  = 13;
 const int MixBlendMode_Color       = 14;
 const int MixBlendMode_Luminosity  = 15;
 
-void main(void) {
-    vec4 Cb = texture(sCacheRGBA8, vUv0);
-    vec4 Cs = texture(sCacheRGBA8, vUv1);
+vec4 brush_fs() {
+    vec2 uv = vUv.xy / vW;
```

Comment:
this can be improved, as discussed on IRC. I'll follow-up with fixes

Reply:
Yep, sounds good - so the basic idea will be to set W=1 in the brush shader code when we want to emulate ```noperspective```, since that isn't available in ES3.0. This can be applied to both ```brush_blend``` and ```brush_mix_blend```.